lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2399,10 +2406,20 @@ LogicalResult StmtEmitter::visitStmt(InstanceOp op) {
 
     // Emit the value as an expression.
     ops.clear();
-    // output ports were lowered to wire
-    if (elt.isOutput())
+
+    // output ports that are not connected to single use output ports were lowered to wire
```

Comment:
Super nit, but please start sentence comments with a capital letter and end with a period.